docs/content/configuration/indexing-service.md

```diff
@@ -216,7 +216,7 @@ its better to write a druid extension module with extending current worker selec
 
 |Property|Description|Default|
 |--------|-----------|-------|
-|`type`|`javascript`.|javascript|
+|`type`|`javascript`.|fillCapacity|
```

Comment:
hmm this seems very odd when placed under the "javascript" strategy. The default "type" for a javascript select strategy is "fillCapacity"?

IMO wording like "required; must be `javascript`" would make more sense.